src/components/common/CompleteLogin.tsx

```diff
@@ -0,0 +1,31 @@
+import Modal from "react-bootstrap/Modal";
+import { LoginButtonWrapper } from "./LoginButtonWrapper";
+import NoChatWizardImg from "images/no-chatbot.png";
+import { Button } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+
+function CompleteProfile() {
+    const { t } = useTranslation();
+
+    return (
+        <>
+            <Modal.Header
+                closeButton
+                className="d-flex align-items-center justify-content-center"
+            >
+                <h3 className="w-100 sub-title text-center">
+                    Complete Login
```

Comment:
move to `translation.json`

Reply:
@pratheeshme can you do this along with the svgs in your ui changes? Currently i have only placed placeholder text